lib/util/host-rules.ts

```diff
@@ -107,8 +108,8 @@ function fromShorterToLongerMatchHost(a: HostRule, b: HostRule): number {
   return a.matchHost.length - b.matchHost.length;
 }
 
-function hostRuleRank({ hostType, matchHost }: HostRule): number {
-  if (hostType && matchHost) {
+function hostRuleRank({ hostType, matchHost, readOnly }: HostRule): number {
+  if ((hostType ?? readOnly) && matchHost) {
```

Comment:
Can you explain this? I don't follow the ?? logic

Reply:
Nah, it's linter thing, I'll turn this off